backend/api/Areas/Research/Controllers/ResearchFileController.cs

```diff
@@ -181,7 +181,7 @@ public IActionResult UpdateResearchFileProperty(long researchFileId, long resear
             }
 
             var researchFilePropertyEntity = _mapper.Map<Dal.Entities.PimsPropertyResearchFile>(researchFilePropertyModel);
-            var researchFile = _pimsService.ResearchFileService.UpdateProperty(researchFileId, researchFilePropertyModel.RowVersion, researchFilePropertyEntity);
+            var researchFile = _pimsService.ResearchFileService.UpdateProperty(researchFileId, researchFilePropertyModel.File.RowVersion, researchFilePropertyEntity);
```

Comment:
nit: if you use the researchFilePropertyEntity  then that extra pass would not be necessary